### PR TITLE
Lesson 6. Create MVC, specs for Trimesters and Mentors

### PR DESCRIPTION
**Pull request by EvgeniiRychkov**
Create MVC, specs for Trimesters and Mentors

### CODE REVIEW

File: spec/requests/mentors_spec.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      before do
+        (1..2).each do |i|
+          Mentor.create!(
+            first_name: "Mentor #{i}",
+            last_name: "Last #{i}",
+            email: "#{i}@gmail.com",
+            max_concurrent_students: 5
+          )
+        end
+      end
```
Comment (JackCSweeney) [inline spec/requests/mentors_spec.rb:15]: Nice setup for this test!

File: spec/requests/trimesters_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        get '/trimesters'
+        expect(response.body).to include('Term 1 2025')
+        expect(response.body).to include('Term 2 2025')
+      end
+    end
+
+    context 'trimesters do not exist' do
+      it 'returns a page with empty list of Trimesters' do
+        get '/trimesters'
+        expect(response.body).not_to include('<li>')
```
Comment (JackCSweeney) [inline spec/requests/trimesters_spec.rb:28]: This is a clever way to test this! Additionally, you could add IDs or classes to the html elements in the view your testing, then with a gem like Capybara, test that those elements don't exist or are empty! This works great as well though.

**Review by JackCSweeney — APPROVED** [review]
